### PR TITLE
Patch input accessibility update

### PR DESCRIPTION
**Pull request by jakerarr**
- updated radio-type/scale-type with WCAG compliant labeling
- Table style update- adds IDs to TH's and aria-labelledby attributes to fields
- Table style update- set default table ID for FormArrayElements if one is not set

### CODE REVIEW

File: src/Forms/FormArrayElement.php [hunk 1]
```diff
@@ -189,6 +189,11 @@ public function getFootFields()
      */
     public function getTableId()
     {
+        //wcag compliance (sets ID for TH and fields)
+        if(empty($this->table_id)){
+            $this->setTableId(rand());
```
Comment (GameCharmer) [inline src/Forms/FormArrayElement.php]: You're probably going to want a character or two in here.  This random is a number only.

File: twig/fields/CKEditor.twig [hunk 2]
```diff
@@ -5,6 +5,9 @@
             style="height:{{ field.height|default('120px') }}"
             id="ck_{{ field.name }}"
             {{ field.getDataAttributeString()|raw }}
+            {% if field.labelledBy != '' %}
```
Comment (GameCharmer) [inline twig/fields/CKEditor.twig]: Replace if blocks with inline ternaries.  Twig creates a ton of code, this'll keep it to a minimum, especially across so many files.
```
{{ field.labelledBy!= '' ? 'aria-labelledby="' ~ field.labelledby ~ '"' }}
```

**Review by GameCharmer — CHANGES_REQUESTED** [review]
Testing general "submit feedback" function

File: src/Forms/FormArrayElement.php (hunk 1 above)
Comment (jakerarr) [reply src/Forms/FormArrayElement.php]: set a prefix of 'table_' before the rand

File: twig/fields/CKEditor.twig (hunk 2 above)
Comment (jakerarr) [reply twig/fields/CKEditor.twig]: Set these to inline ternary! (Did this for all the applicable twig files, not just CKEditor.twig)

File: src/Forms/FormArrayElement.php (hunk 1 above)
Comment (jakerarr) [reply src/Forms/FormArrayElement.php]: Sorry- resolved these on the main page, but it's looking unresolved in the "changes requested" section :P Added 'table_' prefix!

**Comment by GameCharmer** [comment]
Be sure to include your change list in the CHANGELOG.md file under unreleased. :D

File: twig/fields/CKEditor.twig (hunk 2 above)
Comment (jakerarr) [reply twig/fields/CKEditor.twig]: This one also may be a double.  Switched to ternary in all applicable twig files (not just CKEditor.twig)

**Review by GameCharmer — APPROVED** [review]

**Review by GameCharmer — APPROVED** [review]
Changes approved

**Pull request merged**